packages/react-server/package.json

```diff
@@ -28,7 +28,7 @@
     "cookie-parser": "1.3.3",
     "eventemitter3": "0.1.6",
     "express-state": "1.2.0",
-    "lodash": "3.10.1",
+    "lodash": "^4.12.0",
```

Comment:
Just out of curiosity, did we intentionally unpin this? ("Yes" is a reasonable answer, just curious.)